src/source/feeds.rst

```diff
@@ -31,6 +78,635 @@ XML file.
+
+.. index:: media
+.. _feed_media:
+
+media
+"""""
+
+Product images. Media should contain from 0 to N **<image>** elements, where the exact limit depends on the category in taxonomy. **<image>** elements must contain an URL attribute. URL should be a complete link pointing to an image on a publicly available webserver.
+
+Allowed image formats: JPEG, JPG, PNG, GIF, BMP. 
+
+All images will be resized if necessary to a size of maximum 1024px height and 1024px width (preserving the aspect ratio)
+The system will download the images and, if they meet the requirements, store them on our servers in several sizes.
+
+The images will be presented in the order as provided. The first image is shown in search results and as the main image on the item page.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:media>
+                <ad:image url="https://images.pexels.com/photos/62289/yemen-chameleon-chamaeleo-calyptratus-chameleon-reptile-62289.jpeg"/>
+                <ad:image url="https://images.pexels.com/photos/47547/squirrel-animal-cute-rodents-47547.jpeg?auto=compress&amp;cs=tinysrgb&amp;w=1260&amp;h=750&amp;dpr=2"/>
+            <ad:media/>
+======= =========================================================================================================
+
+.. index:: attributes
+.. _feed_attr:
+
+attributes
+""""""""""
+
+Optional collection of ad :ref:`user_defined_attributes` (category-dependent). 
+Note that categories can have mandatory attributes, for which a default value will be filled in if not supplied.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:attributes>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>nl</ad:attributeLocale>
+                    <ad:attributeLabel>Kleu</ad:attributeLabel>
+                    <ad:attributeValue>Rood</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>color</ad:attributeName>
+                    <ad:attributeLocale>en</ad:attributeLocale>
+                    <ad:attributeLabel>Color</ad:attributeLabel>
+                    <ad:attributeValue>Red</ad:attributeValue>
+                </ad:attribute>
+                <ad:attribute>
+                    <ad:attributeName>Model</ad:attributeName>
+                    <ad:attributeValue>Slim</ad:attributeValue>
+                    <ad:attributeValue>Spro</ad:attributeValue>
+                </ad:attribute>
+            </ad:attributes>
+
+======= =========================================================================================================
+
+.. index:: budgetDetails
+.. _feed_budget:
+
+budgetDetails
+"""""""""""""
+
+Section with budget details
```

Comment:
Do you want to explain a bit on what budgets are here or would you want to keep these docs more concise?

Reply:
The idea was that as the budget changes significantly in v5, I would keep it more concise for now, and elaborate more when we upgrade to that version.

Reply:
oki